CONTRIBUTING.md

```diff
@@ -12,7 +14,7 @@ We encourage pull requests and other contributions from the community. Before su
  
 ### Prerequisites
 
-This project should be built against Go 1.16 or newer.
+This project should be built against Go 1.17 or newer.
```

Comment:
It has to be 1.17+ because we're using the `embed` feature to get the version string.